keyboards/takashicompany/minizone/info.json

```diff
@@ -0,0 +1,239 @@
+{
```

Comment:
Please fix the positioning and ordering:
![image](https://user-images.githubusercontent.com/4781841/158282351-e4918215-69ec-4cfe-a462-6f0ccba61da1.png)
![image](https://user-images.githubusercontent.com/4781841/158282389-9aec6320-3320-4507-ba88-b01c6107a0af.png)